pylxd/models/image.py

```diff
@@ -156,6 +156,32 @@ def create_from_simplestreams(cls, client, server, alias,
 
         return client.images.get(op.metadata['fingerprint'])
 
+    @classmethod
+    def create_from_image(cls, client, server, fingerprint=None, alias=None,
+                          public=False, auto_update=False, secret=None,
+                          certificate=None):
+        """Copy an image from remote lxd."""
```

Comment:
Please could we have a docstring comments for the params types.  e.g.

    :param client:  the pylxd client
    :type client: pylxd.Client
    :param server: ...
    :type server: str
    ...etc.